### ISSUE TITLE
MariaDB4j DB mac64 10.3 build is broken due to bintray.com having pulled the plug

### ISSUE THREAD

**Issue by vorburger** [vorburger/MariaDB4j#484]
While looking into picking up #334 prompted by @valdo404 ping, I've run into the following problem while trying to wrap up #451:

```
$ mvn -f DBs/mariaDB4j-db-10.3/pom.xml deploy
(...)
[INFO] mariaDB4j-db-win32 10.3.16 ......................... SUCCESS [  5.741 s]
[INFO] mariaDB4j-db-winx64 10.3.16 ........................ SUCCESS [  3.325 s]
[INFO] mariaDB4j-db-mac64 10.3.16 ......................... FAILURE [  0.555 s]
[INFO] mariaDB4j-db-linux64 10.3.16 ....................... SKIPPED
[INFO] mariaDB4j 10.3 binaries 0.0.0-SNAPSHOT ............. SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  10.057 s
[INFO] Finished at: 2021-11-28T19:08:17+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-antrun-plugin:1.8:run (generate-resources) on project mariaDB4j-db-mac64: An Ant BuildException has occured: The following error occurred while executing this line:
[ERROR] /home/vorburger/dev/MariaDB4j/DBs/mariaDB4j-db-10.3/mariaDB4j-db-mac64-10.3/prepare.xml:21: Can't get https://bintray.com/homebrew/bottles/download_file?file_path=mariadb-10.3.16.mojave.bottle.tar.gz to /home/vorburger/dev/MariaDB4j/tmpDir/input/mariadb-10.3.16.mojave.bottle.tar.gz
[ERROR] around Ant part ...<ant antfile="/home/vorburger/dev/MariaDB4j/DBs/mariaDB4j-db-10.3/mariaDB4j-db-mac64-10.3/prepare.xml"></ant>... @ 8:106 in /home/vorburger/dev/MariaDB4j/DBs/mariaDB4j-db-10.3/mariaDB4j-db-mac64-10.3/target/antrun/build-main.xml
```

For the very short term, in order to unblock, I have 2 choices how to work around this:

A. I could release a mariaDB4j DB v10.3.16 without mac64, and proceed with a MariaDB4j release 2.5.0. I do not want to do that, because I'll bet real hard cryptocurrency that that would cause (valid!) complaints about why "it doesn't work on Mac anymore".

B. I could fix the problem above. I do not want to do that anymore - but welcome contributions from community members who are actively using this project (I'm personally not anymore actually, see also #213) proposing PR for this issue.

C. I can revert the DB version dependency of MariaDB4j Core from DB v10.3.16 back down to v10.3.13. This should unblock the broken build and (hopefully) let me pursue a release, and seems like the most appropriate short term work around here.

**Comment by vorburger** [comment]
>  from DB v10.3.16 back down to v10.3.13

Ah no I had actually never released the v10.3.13 DBs to Maven Central either, back when (and can't anymore now; same problem as above due to the bintray.com story). The last DBs I had published, back on 26-Jan-2018, was ‎10.2.11. 

So the most appropriate short term work around here for me is to go back to that 10.2.11. 

This should unblock a 2.5.0 release. The community can then contribute DB updates.

**Comment by vorburger** [comment]
Three commits related to this were merged as part of #457.

Anyone working on this in the future will want to revert those 3 commits.

@valdo404 would you be interested in contributing a PR with for this?

The workaround fixed #451 and enables pursuing #334.

PS: Please [Sponsor my maintenance this project](https://github.com/sponsors/vorburger)! It's purely symbolic - but I would appreciate it.

**Comment by vorburger** [comment]
Anyone interested in fixing this would probably consider straight looking into #344 to add latest binaries for all platforms, instead of fixing 10.3.16?
